### DISCUSSION TITLE
How to Lure enemies without teleporting them? What code needs to be turned off?

### DISCUSSION THREAD

**Discussion by Misaka-Mikoto-G** [winstxnhdw/lc-hax#252] [General]
How to Lure enemies without teleporting them? What code needs to be turned off?

**Comment by winstxnhdw** [comment]
It already does that

**Reply by Misaka-Mikoto-G** [reply]
no,sometime will teleport Snare Flea or Invisible Nutcracker，Even dead Snare Flea can cause player items drop.

**Reply by winstxnhdw** [reply]
Doesn't do that anymore. Make sure you are on latest.

**Reply by Misaka-Mikoto-G** [reply]
ok

**Reply by winstxnhdw** [reply]
IIRC, `M3` will still teleport Snare Flea. You have to use `/hate` directly.

**Reply by Misaka-Mikoto-G** [reply]
ok,btw How do I disable unwanted Patches

**Reply by winstxnhdw** [reply]
Literally delete them. The code is very nicely splitted. Removing the patches won't break the game.

**Reply by Misaka-Mikoto-G** [reply]
ok thanks

**Reply by Misaka-Mikoto-G** [reply]
if i don't need M3 teleport Snare Flea,which codes need to be turned off?

**Reply by winstxnhdw** [reply]
Look at the TriggerMod file

**Reply by Misaka-Mikoto-G** [reply]
/*if (collider.TryGetComponent(out PlayerControllerB player)) {
                Helper.FindObject<CentipedeAI>()?.ClingToPlayerServerRpc(player.playerClientId);
                this.PromptEnemiesToTarget(player, this.FunnyReviveEnabled)
                    .ForEach(enemy => Chat.Print($"{enemy} prompted!"));
                break;*/
            }

is this right?

**Reply by winstxnhdw** [reply]
the CentipedeAI one

**Reply by Misaka-Mikoto-G** [reply]
and Dead centipedes will give lured person centipede effects without the actual centipede this one can be disabled?

**Reply by winstxnhdw** [reply]
That'll all be gone when you disable Centipede in TriggerMod

**Reply by Misaka-Mikoto-G** [reply]
ok thanks

**Reply by Misaka-Mikoto-G** [reply]
still teleport Invisible Nutcracker im sure it on latest.

**Reply by winstxnhdw** [reply]
What is invisible nutcracker?

**Reply by Misaka-Mikoto-G** [reply]
m3 will teleport a Invisible Nutcracker kill player,i can't see it but other player can see a dead Nutcracker use his gun

**Reply by winstxnhdw** [reply]
Can you comment out the line with `AimGunServerRpc` in `IEnemyPrompter.cs` and see if the issue stops?

**Reply by Misaka-Mikoto-G** [reply]
ok will do
